ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1075,18 +1075,21 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                           customAccessoryView: UIView? = nil,
                           accessoryType: TableViewCellAccessoryType = .none) {
         if let attributedTitle = attributedTitle {
-            titleLabel.attributedText = attributedTitle
+            self.attributedTitle = attributedTitle
```

Comment:
In the attributed string scenario the text property is not set.
There are other places using this property for layout sizing calculation:

https://github.com/microsoft/fluentui-apple/blob/261f2b0fe5721ff6e5d3ffa26a64de755422dde7/ios/FluentUI/Table%20View/TableViewCell.swift#L770

Have you tested an attributed string with a larger font to see how this could impact the height and width of the cell? If the height is not correctly calculated, we need to have a bug filed for that.

Reply:
Screenshot below is with larger text on and set to max
![Simulator Screen Shot - iPhone 12 Pro - 2022-07-06 at 16 58 43](https://user-images.githubusercontent.com/22566866/177661915-c4644aa0-aa5a-493a-876b-4a60e238fe5b.png)